spec/openapi.yaml

```diff
@@ -626,135 +645,190 @@ components:
           example: 1648038843
-        ro:
-          type: boolean
-          example: False
-        metadata:
-          type: object
-          description: Print file metadata parsed from G-code, all data are optional
+        m_timestamp:
+          description: Timestamp in seconds
+          type: integer
+          example: 1648042843
+
+    PrintFileInfo:
+      allOf:
+        - $ref: '#/components/schemas/FileInfo'
+        - type: object
           properties:
-            bed_temperature:
-              type: string
-              description: Degrees Celsius
-              example: "60"
-            nozzle_temperature:
-              type: string
-              description: Degrees Celsius
-              example: "215"
-            brim_width:
-              type: string
-              description: Milimeters
-              example: "0"
-            estimated printing time (normal mode):
-              type: string
-              example: "42m 42s"
-            faded_layers:
-              type: string
-              example: "42"
-            filament cost:
-              type: string
-              example: "0.57"
-            filament used [cm3]:
-              type: string
-              example: "12.44"
-            filament used [g]:
-              type: string
-              example: "15.79"
-            filament used [mm]:
+            display_name:
               type: string
-              example: "5170.06"
-            filament_type:
+              description: Long Filename
+              example: "Spice_Harvester_0.3mm_PLA_MK3S_12m.gcode"
+            display_path:
               type: string
-              example: "PLA"
-            fill_density:
-              type: string
-              description: Percent
-              example: "20%"
-            initial_exposure_time:
-              type: string
-              description: Seconds
-              example: "5"
-            layer_height:
-              type: string
-              description: Milimeters
-              example: "0.3"
-            material_name:
-              type: string
-              example: "PLA Sandstorm Orange"
-            exposure_time:
-              type: string
-              description: Seconds
-              example: "2"
-            max_exposure_time:
-              type: string
-              description: Seconds
-              example: "3"
-            max_initial_exposure_time:
-              type: string
-              description: Seconds
-              example: "3"
-            min_exposure_time:
-              type: string
-              description: Seconds
-              example: "1"
-            min_initial_exposure_time:
-              type: string
-              description: Seconds
-              example: "1"
-            nozzle_diameter:
-              type: string
-              description: Milimeters
-              example: "0.4"
-            print_time:
+              example: "/PrusaLink gcodes/Dune"
```

Comment:
Are we expected to provide both long filename and _decode the whole prefix_ too, for each file? :-O

Reply:
We want to show nice (display) path for user on File detail. And i'm not sure how difficult is, to compute that display_path from previews request in file listing. May be display_path can be optional, which actually is. 

Please figure out @Majkluss and @Lowentwickler

Reply:
Doesn't the server already have all the long path segments of the folders above? Is it really relying on the printer doing all that? 😇

Reply:
I construct path manually from display_name. There is no need for display path. And for the operations I use paths from `refs`